boards/xtensa/esp32/common/src/esp32_es8388.c

```diff
@@ -179,6 +177,35 @@ int esp32_es8388_initialize(int i2c_port, uint8_t i2c_addr, int i2c_freq,
           goto errout;
         }
 
+      /* Now we can use this I2S interface to initialize the ES8388 input
+       * which will return an audio interface.
+       */
+
+      g_es8388_lower[1].address   = i2c_addr;
+      g_es8388_lower[1].frequency = i2c_freq;
+
+      es8388 = es8388_initialize(i2c, i2s, &g_es8388_lower[1]);
+      if (es8388 == NULL)
+        {
+          auderr("ERROR: Failed to initialize the ES8388\n");
+          ret = -ENODEV;
+          goto errout;
+        }
+
+      /* Create a device name */
+
+      snprintf(devname, 12, "pcm_in%d",  i2s_port);
```

Comment:
```suggestion
      snprintf(devname, sizeof(devname), "pcm_in%d",  i2s_port);
```
Here and similar places